libs/utils/logWebVitals.js

```diff
@@ -62,14 +62,35 @@ function getElementInfo(el) {
   return `${el.outerHTML.substring(0, 100)}...`;
 }
 
-function observeLCP(lanaData, delay) {
+function isFragmentFromMep(fragPath, mep) {
+  return mep.experiments?.some(({ selectedVariant }) => {
+    const { commands = [], fragments = [] } = selectedVariant || {};
+
+    return commands.some((cmd) => {
+      try {
+        return new URL(cmd.target).pathname === fragPath;
+      } catch {
+        return false;
+      }
+    }) || fragments.some((cmd) => cmd?.val === fragPath);
+  });
+}
+
+function observeLCP(lanaData, delay, mep) {
   new PerformanceObserver((list) => {
     const entries = list.getEntries();
     const lastEntry = entries[entries.length - 1]; // Use the latest LCP candidate
     lanaData.lcp = parseInt(lastEntry.startTime, 10);
     const lcpEl = lastEntry.element;
     lanaData.lcpElType = lcpEl.nodeName.toLowerCase();
     lanaData.lcpEl = getElementInfo(lcpEl);
+    const closestFrag = lcpEl.closest('.fragment');
+    lanaData.isFrag = closestFrag ? 'true' : 'false';
+    if (closestFrag) {
+      lanaData.isMep = isFragmentFromMep(closestFrag.dataset.path, mep) ? 'true' : 'false';
+    } else {
+      lanaData.isMep = 'false';
+    }
```

Comment:
Nit: couldn't we also do:
```suggestion
    lanaData.isMep = closestFrag && isFragmentFromMep(closestFrag.dataset.path, mep)
      ? 'true' : 'false';
```